test/functional/wallet_backwards_compatibility.py

```diff
@@ -178,18 +178,25 @@ def run_test(self):
         node_master_wallets_dir = node_master.wallets_path
         node_v19_wallets_dir = node_v19.wallets_path
         node_v17_wallets_dir = node_v17.wallets_path
-        node_v16_wallets_dir = node_v16.chain_path
+        node_v16_wallets_dir = node_v16.wallets_path
         node_master.unloadwallet("w1")
         node_master.unloadwallet("w2")
         node_master.unloadwallet("w3")
 
         for node in legacy_nodes:
             # Copy wallets to previous version
             for wallet in os.listdir(node_master_wallets_dir):
-                shutil.copytree(
-                    os.path.join(node_master_wallets_dir, wallet),
-                    os.path.join(self.nodes_wallet_dir(node), wallet)
```

Comment:
In df850c25:
Removed the only usage of the `nodes_wallet_dir` function but not the, now unused, function.

Reply:
Done